docs/dxp/7.x/en/installation-and-upgrades/installing-liferay-dxp-on-premises/installing-liferay-on-an-application-server/installing-liferay-on-weblogic.md

```diff
@@ -0,0 +1,211 @@
+# Installing DXP on WebLogic
+
+Installing Liferay DXP on WebLogic requires deploying dependencies, modifying configuration files, configuring a database and a mail session, and deploying the DXP WAR file on a WebLogic server.
+
+It is **highly recommended** to install web apps, including DXP, in a WebLogic Managed server. Deploying to a Managed Server lets you to start or shut down DXP more quickly and facilitates transitioning into a cluster configuration. This article therefore focuses on installing DXP in a Managed Server.
+
+Before getting started, create the Admin and Managed Servers. See [WebLogic's documentation](http://www.oracle.com/technetwork/middleware/weblogic/documentation/index.html) for instructions on setting up and configuring Admin and Managed Servers.
+
+Liferay DXP requires Java JDK 8 or 11. Please consult the [Liferay DXP Compatibility Matrix](https://web.liferay.com/documents/14/21598941/Liferay+DXP+7.2+Compatibility+Matrix/b6e0f064-db31-49b4-8317-a29d1d76abf7?) to learn more about the supported environments.
+
+To install Liferay DXP on WebLogic, download the following from the [Liferay Community Downloads site](https://www.liferay.com/downloads-community):
+
+* DXP WAR file
+* Dependencies ZIP file
+* OSGi Dependencies ZIP file
+
+Enterprise subscribers can download the files from [Help Center](https://customer.liferay.com/downloads). Subscribers also need an order number or activation key for Liferay DXP. See the [Activating Liferay DXP](../../02-setting-up-liferay-dxp/activating-liferay-dxp.md) article.
+
+Here are the basic steps for installing DXP on WebLogic:
+
+1. [Configuring WebLogic for DXP](#configuring-weblogic)
+1. [Declaring the `Liferay Home` folder](#declaring-the-liferay-home-folder)
+1. [Installing the dependencies](#installing-dxp-dependencies)
+1. [Deploying the WAR](#deploying-dxp)
+
+## Configuring WebLogic
+
+### Configuring WebLogic's Node Manager
+
+WebLogic requires a Node Manager to start and stop managed servers. Before installing DXP, configure the Node Manager included with the WebLogic installation. This is set in the `domains/your_domain_name/nodemanager/nodemanager.properties` file. Open this file and set the `SecureListener` property to `false`:
+
+```properties
+SecureListener=false
+```
+
+This setting disables the encryption (SSL) requirement for the Node Manager, allowing it to accept unencrypted connections. Although it's possible to run DXP with this property set to `true`, there may be some difficulties doing so. Furthermore, if `SecureListener` set to `true`, then configure the machine in the Admin Server's console to accept unencrypted connections from the Node Manager. To do this:
```

Comment:
@justinchoi001 I find this confusing. Is the last part of this paragraph (and the steps below) about configuring the Admin Server to accept unencrypted connections? Do you only need to do these steps if you're setting `SecureListener=true`?